src/screens/OtherProfile/index.js

```diff
@@ -94,10 +91,14 @@ const OtherProfile = () => {
     isBlocking,
     isLoading,
     isProfileFetching,
+    isCurrentFollowed,
+    otherUserId: other_id,
+    selfUserId: user_id,
 
     refetchBlockStatus,
     refetchOtherProfile,
-    setOtherProfileData: setDataMain
+    setOtherProfileData: setDataMain,
+    setIsCurrentFollowed
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `isCurrentFollowed`, `otherUserId: other_id`, and `selfUserId: user_id` to the destructuring of `useOtherProfileScreenHooks()` seems to be a good change for clarity and readability. It also appears that these new state variables are replacing the previously locally managed states `user_id`, `username`, and `other_id`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->